about/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About | Doodle Dandy | Doodle Dandy</title>
```

Comment:
You don't need `| Doodle Dandy` twice.
```suggestion
    <title>About | Doodle Dandy</title>
```

Reply:
This is still not fixed